lib/sycamore/sycamore/config.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+from typing import Optional, Any
+
+from sycamore.llms import LLM
+
+
+@dataclass
+class Config:
+
+    opensearch_client_config: Optional[dict[str, Any]] = None
+    opensearch_index_name: Optional[str] = None
+    opensearch_index_settings: Optional[dict[str, Any]] = None
+
+    llm: Optional[LLM] = None
```

Comment:
So if I'm a user and I want to set my own config vars how do I do that? Can I just do something like:
```python
cfg = Config()
cfg.myfieldwithlikegcscredentialsorsomething = "efngsogijaeropgijw"
```
or do I have to pull subclassing shenanigans